js/client-chat.js

```diff
@@ -1738,7 +1738,10 @@
 			} else if (mayNotify && this.id.substr(0, 5) === 'help-') {
 				this.notifyOnce("Help message from " + name, "\"" + message + "\"", 'pm');
 			} else if (mayNotify && name !== '~') { // |c:|~| prefixes a system message
-				this.subtleNotifyOnce();
+				var mutedChats = Dex.prefs('mutedchats') || {};
```

Comment:
I would call this "muted rooms" or "muted panels". I think that would be much less confusing than "muted chats", considering what the rest of the codebase calls them.